packages/react-menu/src/components/Menu/useMenu.tsx

```diff
@@ -47,7 +52,13 @@ export const useMenu = (props: MenuProps, ref: React.Ref<HTMLElement>, defaultPr
   });
 
   state.menuPopup.children = (Component, p) => {
-    return <Component {...p}> {state.menuList} </Component>;
+    return (
+      // eslint-disable-next-line @typescript-eslint/ban-ts-comment
+      // @ts-ignore
```

Comment:
Why is ts-ignore needed? Can you cast something instead to avoid this "nuclear option" of disabling all checking for the next line?

Reply:
Just wondering, will be there a TS error if you will use `React.createElement` instead of JSX (`React.createElement(Component, p, state.menuList)`)?

Reply:
used `createElement` got rid of those pesky ts errors